test/e2e/chainsaw/check-rgd-deletion/ignore-chainsaw-test.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: IgnoreTest
+metadata:
+  name: check-rgd-deletion
+spec:
+  description: | 
+    Tests if deleting a ResourceGraphDefinition deletes the Instance CRD
+  steps:
+  - name: install-rgd
+    try:
+    #- description: Install the RGD that creates an Instance CRD
+    - apply:
+        file: rgd.yaml
+      description: Apply RGD
+    - assert:
+        file: rgd-assert.yaml
+      description: Verify RGD state
+    - assert:
+        file: instancecrd-assert.yaml
+      description: Verify Instance CRD state
+    catch:
+    - description: kro controller pod logs collector
+      podLogs:
+        selector: app.kubernetes.io/instance=kro
+        namespace: kro-system
+    finally:
+    - description: sleep operation
+      sleep:
+        duration: 5s
+  - name: delete-rgd
+    try:
+    #- description: Delete the RGD that creates an Instance CRD
+    - delete:
+        ref:
+          apiVersion: kro.run/v1alpha1
+          kind: ResourceGraphDefinition
+          name: simple-deployment-for-deletion-rgd
+          namespace: default
+      description: Delete RGD
+    catch:
+    - description: kro controller pod logs collector
+      podLogs:
+        selector: app.kubernetes.io/instance=kro
+        namespace: kro-system
```

Comment:
This step deletes the RGD. Now, we need to check if its CRD is deleted as well, so we need to add another step as follows:
```
  - name: check-that-crd-is-deleted
    try:
    - error:
        file: instancecrd-assert.yaml
```